packages/mui-base/src/useAutocomplete/useAutocomplete.d.ts

```diff
@@ -66,6 +66,9 @@ export interface UseAutocompleteProps<
    * If `true`, the selected option becomes the value of the input
    * when the Autocomplete loses focus unless the user chooses
    * a different option or changes the character string in the input.
+   *
+   * In addition of `freeSolo` mode, the typed value will be the input value
```

Comment:
I don't think I understand what this is saying. Is it meant to be "In addition _to_ freeSolo mode", or "When using freeSolo mode"?